### ISSUE TITLE
Use AFNetworking

### ISSUE THREAD

**Issue by mattgreen** [mattgreen/elevate#6]
Maintaining the HTTP component requires more effort than it's worth. Find a way to connive AFNetworking to play nicely with Elevate.

**Comment by matthewsinclair** [comment]
Really? I've just spent two days ripping out AFNetworking to use Elevate because of the way AFNetworking doesn't play very nicely with the threads it runs on. If you do this, can you make sure that the sync behaviour of your current HTTTPClient is retained?

**Comment by mattgreen** [comment]
Of course! The whole point of Elevate is synchronous-cancellable I/O. I'd just rather work on other parts of it beyond HTTP.
